src/pages/EditOrderPage.js

```diff
@@ -169,6 +174,16 @@ const EditOrderPage = () => {
     setDeliveryDate(parseDateToEdit(order.deliveryDate));
     setDeliveryMethod(order.deliveryMethod);
 
+    if (order.deliveryFee > 0) {
+      setDeliveryPrice(order.deliveryFee);
+    } else {
+      setDeliveryPrice(orderDeliveryFee);
+    }
```

Comment:
Sometimes I avoid ifs when I see something that repeats inside the if, does it make sense? Totally optional, just showing how I would do it

```suggestion
      setDeliveryPrice(order.deliveryFee > 0 ? order.deliveryFee : orderDeliveryFee);
```

Reply:
Yes much better!!!